docs/guide/setup/chemical_systems_and_thermodynamic_cycles.rst

```diff
@@ -1,4 +1,71 @@
-ChemicalSystems and thermodynamic cycles
-========================================
+.. _userguide_chemicalsystems_and_components:
 
-issue #759
\ No newline at end of file
+ChemicalSystems, Components and Thermodynamic Cycles
+====================================================
+
+Chemical Systems
+----------------
+
+In order to define the input systems to a Protocol, so as the end states of an alchemical transformation,
```

Comment:
Apologies, the "so as the end states of the alchemical transformation" part of this sentence doesn't really make sense to me - I think I know what you're trying to say, but the sentence may need restructuring?